responsibleai/responsibleai/databalanceanalysis/aggregate_balance_measures.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from typing import Callable, Dict, List
+
+import numpy as np
+import pandas as pd
+
+from responsibleai.databalanceanalysis.balance_measures import BalanceMeasures
+from responsibleai.databalanceanalysis.constants import Measures
+
+
+class _MeasureHelper:
+    """
+    Helper to compute each of the individual aggregate balance measures.
+    """
+
+    @staticmethod
+    def _get_generalized_entropy_index(
```

Comment:
this seems like something that should eventually live in the fairlearn repository (https://github.com/fairlearn/fairlearn)